maquetado/login.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <div class="login">
+      <img class="logo" src="./assets/LogoWolox.png" alt="wolox-logo">
+      <label for="email" class="text-from">Email</label>
+      <input type="text" id="email" class="input-form">
+      <label for="password" class="text-from">Password</label>
+      <input type="password" id="password" class="input-form">
+      <a href="index.html">
+        <button class="login-button text-from">Login</button>
+      </a>
+      <hr class="splitter">
```

Comment:
I think you can remove this item and use the border-bottom of the tag above

Reply:
I used `hr` tag because if in the future we had to move the buttons the separator should be in the same place